src/vivarium_public_health/disease/special_disease.py

```diff
@@ -155,6 +177,42 @@ def on_time_step(self, event):
 
         self.population_view.update(pop)
 
+    def compute_disability_weight(self, index):
```

Comment:
These are moved and renamed.  Behavior has changed as well.  Now only living people will have non-zero disability/excess mortality rates.  This should prevent broad categories of errors we see periodically.